src/segment-loader.js

```diff
@@ -349,12 +348,25 @@ export default class SegmentLoader extends videojs.EventTarget {
   }
 
   /**
+   * (re-)schedule monitorBufferTick_ to run as soon as possible
+   *
+   * @private
+   */
+  monitorBuffer_() {
```

Comment:
The real difference is in how long in the future things are scheduled. Calling `monitorBuffer_` forces the tick to happen next (it either schedules or reschedules the timer tick to be just one millisecond). Whereas `monitorBufferTick_` always schedules the next time to be 500ms in the future.

Originally, I had a piece of code with an optional parameter to `monitorBuffer_` to set/reset different timer durations in one function but I decided that was "too clever".